test-cases/longevity/longevity-cdc-3d-400gb.yaml

```diff
@@ -0,0 +1,30 @@
+test_duration: 4550
+
+prepare_write_cmd: ["cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb.yaml ops'(insert=1)' cl=QUORUM n=50000000 -port jmx=6868 -mode cql3 native -rate threads=100",
+                    "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_preimage.yaml ops'(insert=1)' cl=QUORUM n=50000000 -port jmx=6868 -mode cql3 native -rate threads=200",
+                    "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_postimage.yaml ops'(insert=1)' cl=QUORUM n=50000000 -port jmx=6868 -mode cql3 native -rate threads=200",
+                    "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_preimage_postimage.yaml ops'(insert=1)' cl=QUORUM n=50000000 -port jmx=6868 -mode cql3 native -rate threads=200"
+                   ]
+
+stress_cmd: [ "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,update_all=2,delete1=1)' cl=QUORUM duration=4320m -port jmx=6868 -mode cql3 native -rate threads=100",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_preimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,update_all=2,delete1=1)' cl=QUORUM duration=4320m -port jmx=6868 -mode cql3 native -rate threads=100",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_postimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,update_all=2,delete1=1)' cl=QUORUM duration=4320m -port jmx=6868 -mode cql3 native -rate threads=100",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_400gb_preimage_postimage.yaml ops'(insert=2,read1=1,update_number=1,update_all=2,update_name=1,delete1=1)' cl=QUORUM duration=4320m -port jmx=6868 -mode cql3 native -rate threads=100"
+             ]
+
+n_db_nodes: 6
+n_loaders: 4
+n_monitor_nodes: 1
+
+round_robin: true
+
+instance_type_db: 'i3.4xlarge'
```

Comment:
Does it have enough space?

Reply:
Monitor shows that total space is 4TB. I think it should be enough. Will run for 3days.